hw3/01-characters.js

```diff
@@ -0,0 +1,43 @@
+// url for the Thrones API
+const url = 'https://thronesapi.com/api/v2/Characters';
+
+fetch(url)
+.then(response => response.json())
+.then(characters => {
+    const characterList = document.getElementById('characters');
+    
+    characters.forEach(character => {
+      const { firstName, lastName, imageUrl, title } = character;
+
+      const characterContainer = document.createElement('section');
+      characterContainer.classList.add('character-container');
+
+      const characterImage = document.createElement('img');
+      characterImage.src = imageUrl;
+      characterImage.alt = imageUrl;
+
+      const characterName = document.createElement('h2');
+      characterName.textContent = `${firstName} ${lastName}`;
+
+      const characterTitle = document.createElement('h3');
+      characterTitle.textContent = title;
```

Comment:
Consider using the figure and figcaption elements, this exercise would be a good use case for them.